Readme.md

```diff
@@ -1,113 +1,35 @@
-Locate the <strong>viewSitePanel </strong>control and change its <strong>Dock </strong>property from <strong>Fill </strong>to <strong>None</strong>.<br />
-<img src="https://raw.githubusercontent.com/DevExpress-Examples/how-to-create-information-panels-e2690/15.2.5+/media/1926b8c0-b588-484a-a635-f758d3477f9f.png"></p><p>To ensure that the Template can be correctly resized, set the <strong>SplitContainerControl</strong>’s <strong>Dock </strong>property to <strong>Fill </strong>and the <strong>FixedPanel </strong>property to <strong>Panel2</strong>.<br />
-<img src="https://raw.githubusercontent.com/DevExpress-Examples/how-to-create-information-panels-e2690/15.2.5+/media/6b77e4c5-7218-43ab-8b63-e8d8a1a6ec5e.png"><br />
-To be able to fill the information panel with data, the <strong>SplitContainerControl </strong>must be externally visible. For this purpose, define an interface via which the information panel will be accessed. This will decouple data-supplying Controllers from particular Template types. Instead they will rely on the interface.</p><p></p>
+To accomplish this task, follow the steps below: 
 
-```cs
-public interface IInfoPanelTemplateWin : IFrameTemplate {        
-    DevExpress.XtraEditors.SplitContainerControl SplitContainer { get; }
-}
+1. Create a custom template as described in the following topic: [Templates](https://docs.devexpress.com/eXpressAppFramework/112609/ui-construction/templates).
+2. Add to this custom template a UI element that should display the additional information (e.g. [div](https://www.w3schools.com/tags/tag_div.ASP)).
+3. To pass data to this element, use a dedicated [ViewController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ViewController) that accesses the template and passes required information to this template.
 
+> [!WARNING]
+> We created this example for demonstration purposes and didn't intend to address all possible usage scenarios with it.
+> If this example doesn't have some functionality or you wish to change its behavior, feel free to extend this example as you need. Please note that this is quite a complex task, that requires good knowledge of XAF: [UI Customization Categories by Skill Level](https://www.devexpress.com/products/net/application_framework/xaf-considerations-for-newcomers.xml#ui-customization-categories). You will likely need to research how our components work under the hood. Refer to the following help topic for more information: [Debug DevExpress .NET Source Code with PDB Symbols](https://docs.devexpress.com/GeneralInformation/403656/support-debug-troubleshooting/debug-controls-with-debug-symbols).
```

Comment:
```suggestion
> If this example does not have certain functionality or you want to change its behavior, you can extend this example as needed. Please note that this can be a complex task that requires good knowledge of XAF: [UI Customization Categories by Skill Level](https://www.devexpress.com/products/net/application_framework/xaf-considerations-for-newcomers.xml#ui-customization-categories). You will likely need to research how our components work under the hood. Refer to the following help topic for more information: [Debug DevExpress .NET Source Code with PDB Symbols](https://docs.devexpress.com/GeneralInformation/403656/support-debug-troubleshooting/debug-controls-with-debug-symbols).
```